storage.go

```diff
@@ -50,6 +51,9 @@ var DefaultStorage = &Storage{
 	},
 }
 
+// LocalCache is an optional in memory cache that is consulted before memcache
+var LocalCache *lru.ARCCache
```

Comment:
Feels weird to use a global var which will be shared among potentially many Storage instances.